cypress/support/commands.ts

```diff
@@ -821,8 +821,8 @@ const setScriptToLanguage = (
 ) => {
   return cy.isIoxOrg().then(isIox => {
     if (lang === 'influxql') {
-      // give cypress some time to turn on the feature flag `influxqlUI`
-      // this block can be removed after this feature flag is removed
+      // give cypress some time to set up
+      // database and retetion policy (DBRP) mappings
```

Comment:
minor nit - retention policy

Reply:
Nice catch! Fixed in https://github.com/influxdata/ui/pull/6809/commits/dbd487c18a083ae691c2a33d99ed0e693d667207